app/services/jobs/return-logs/create-quarterly-return-cycles.service.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+/**
+ * Given a return cycle, generate each quarterly return period
+ * @module CreateQuarterlyReturnCycleService
+ */
+
+const { determineReturnsPeriods } = require('../../../lib/return-periods.lib.js')
+
+/**
+ * Given a return cycle, generate each quarterly return period
+ *
+ * @param {object} returnCycle - the return cycle to break into quarterly period
+ *
+ * @returns {Array<object>} the created quarterly return periods
+ */
+function go(returnCycle) {
+  if (!returnCycle) {
+    return []
+  }
+
+  // To reuse as much code as possible we use two dates to determine the quarterly return periods
+  // The first one uses the start date of the return cycle, and the second uses the end date
+  // This enables us to get the correct year for quarter four while reusing exisitng code
+  const quarterlyReturnPeriods = determineReturnsPeriods(returnCycle.startDate)
+  const quarterFourReturnPeriod = determineReturnsPeriods(returnCycle.endDate)
```

Comment:
I wonder, if we moved this function to `app/lib/return-periods.lib.js` would that actually make it even easier/clearer?

Also, it means you can ignore my file change suggestion!